extensions/objects/templates/algebraic-expression-editor.component.ts

```diff
@@ -56,7 +58,7 @@ export class AlgebraicExpressionEditorComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    if (this.value === null) {
+    if (this.value === (null || undefined)) {
```

Comment:
This seems to be a bit counterintuitive why not. Also why this can now be `undefined`?
```suggestion
    if (this.value === null || this.value === undefined) {
```

Reply:
done

Reply:
Also why this can now be undefined?

Reply:
We're basically assigning `' '` for all invalid values. Since this is an @Input property it can be undefined after component 
 interactions. Already added a comment for this at top of `@Input() value !: string`

Reply:
Still not really sure what you mean.

Reply:
Sorry for not making it clear. The property `value` is undefined at the top, we're later assigning it to property `currentValue` which is of type string. We can't assign an undefined value to property of type string therefore we need to add a check here that if value is undefined then initialize it with `''`. Does that sg ?

Reply:
Can we initialize it to `''` at the top then?

Reply:
done

Reply:
Thanks!